src/modules/file-share/file-share-list/file-share-list.html

```diff
@@ -79,14 +79,14 @@ <h5 ng-if="fileShareCntrl.fileShareList.length !== 0">{{fileShareCntrl.filteredF
         </center>
         <div class="list-group list-view-pf list-view-pf-equalized-column" ng-repeat="fileShare in fileShareCntrl.filteredFileShareList=(fileShareCntrl.fileShareList | filter: fileShareCntrl.searchBy | orderBy : fileShareCntrl.ascOrder + fileShareCntrl.orderBy)">
             <div class="list-group-item">
-                <div class="list-view-pf-left col-md-1">
+                <!-- <div class="list-view-pf-left col-md-1">
                   <i class="fa fa-angle-right" aria-hidden="true"></i>
-                </div>
+                </div> -->
                 <div class="list-view-pf-left col-md-1">
                     <span title="{{fileShare.status}}" ng-if="fileShare.status !=='Stopped'" class="pficon pficon-ok list-view-pf-icon-success"></span>
-                    <span title="{{fileShare.status}}" ng-if="fileShare.status ==='Stopped'" class="fa fa-arrow-circle-o-down"></span>
+                    <span title="{{fileShare.status}}" ng-if="fileShare.status ==='Stopped'" class="fa fa-arrow-circle-o-down list-view-icon-failure"></span>
```

Comment:
why this css is required - "list-view-icon-failure" . i think "fa fa-arrow-circle-o-down"  will take care for red color icons